visualization/src/pcl_visualizer.cpp

```diff
@@ -2980,10 +2980,10 @@ pcl::visualization::PCLVisualizer::updateColorHandlerIndex (const std::string &i
     return (false);
   }
 
-  int color_handler_size = int (am_it->second.color_handlers.size ());
-  if (index >= color_handler_size)
+  size_t color_handler_size = am_it->second.color_handlers.size ();
+  if (!(size_t(index) < color_handler_size))
```

Comment:
Sorry, missing space again :(

Reply:
which space needs to be there? `size_t (index)`? something about brackets?

Reply:
`size_t (index)`

Reply:
fixed